src/screens/ENSAssignRecordsSheet.tsx

```diff
@@ -122,6 +122,10 @@ export default function ENSAssignRecordsSheet() {
     ? BottomActionHeightSmall
     : BottomActionHeight;
 
+  useEffect(() => {
+    setAvatarUrl('');
```

Comment:
I am not sure about this solution – my main concern is that, why isn't it taking the color of `initialAvatarUrl` as we are initializing the `avatarUrl` state on line 112?

Reply:
hmm good q, looks like somehow avatarUrl persists even when dismissing the sheet